examples/image_classification/mlflow_image_classification_peft_lora.py

```diff
@@ -17,28 +17,29 @@
 import mlflow
 import os
 
+import sys
+sys.path.append('../')
```

Comment:
This looks like a hack to get something done. What do you what to achieve by this?